src/vunnel/providers/alma/parser.py

```diff
@@ -58,8 +58,59 @@ def _normalize(self, vuln_entry: dict[str, Any], version: str) -> tuple[str, str
         vuln_id = vuln_entry["id"]
         vuln_schema = vuln_entry.get("schema_version", "1.7.0")  # TODO: this is a bit of a hack;
         # we should see whether upstream is willing to put schema versions in the data.
+
+        # Parse modularity from summary field
+        rpm_modularity = None
+        summary = vuln_entry.get("summary", "")
+        if summary:
+            rpm_modularity = self._parse_modularity_from_summary(summary)
+
+        # Add modularity to affected packages if found
+        if rpm_modularity and "affected" in vuln_entry:
+            for affected_pkg in vuln_entry["affected"]:
+                if "ecosystem_specific" not in affected_pkg:
+                    affected_pkg["ecosystem_specific"] = {}
+                affected_pkg["ecosystem_specific"]["rpm_modularity"] = rpm_modularity
+
+        # Add vunnel-specific metadata to indicate this is an advisory record
+        if "database_specific" not in vuln_entry:
+            vuln_entry["database_specific"] = {}
+        if "vunnel" not in vuln_entry["database_specific"]:
+            vuln_entry["database_specific"]["vunnel"] = {}
+        vuln_entry["database_specific"]["vunnel"]["record_type"] = "advisory"
```

Comment:
`advisory` tells grype-db to emit unaffected package handles instead of affected package handles. Consider making this more explicit.